lib/lrama/trace.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
What is the reason why keeping `Trace` module even so introducing `Tracer`?

Reply:
The discomfort of the call made it so. For example, the `Trace` class would be `Trace.trace` or `Trace.run`, but `Tracer.trace` or `Tracer.run` seemed more natural. What do you think?

Reply:
My concern is that classes under `class Trace` are used only by `Tracer` then it's easy to understand if these classes are placed within `Tracer`.

Reply:
refixed: https://github.com/ruby/lrama/pull/534/commits/4026c1dc1451f9d7ac7bf03838259d8e7edd4d0e